### DISCUSSION TITLE
Sublist and fragments

### DISCUSSION THREAD

**Discussion by ced75** [quarto-dev/quarto-cli#3269] [Q&A]
Hi,

When I try to display a list with sub-items in different fragments, each fragment displays a new list (instead of the sub-items).
Example without fragments (the result I'm looking for):
```
- item 1  
    - subitem 1  
    - subitem 2  
        - subsubitem 1  
        - subsubitem 2  
```

Now, if I insert 3 dots between subitems (for example between subitem1 and subitem2), I get the following result
```
- item 1  
    - subitem 1  
- subitem 2  
    - subsubitem 1  
    - subsubitem 2  
```

I get the same with :::  {.fragment}

Is there a way to displays subitems within different fragments?

Thanks for your help

**Comment by frukto** [comment]
I am not sure what you are looking for, but the following reveals subitems one after another. Though I don't know how to fragment subsubitems in this manner.
 
```
- item 1

  ::: {.fragment fragment-index=1}
  - subitem 1
  :::
  ::: {.fragment fragment-index=2}
  - subitem 2
  :::
```
